CharacterCreation/02_Classes.txt

```diff
@@ -188,17 +187,32 @@ you're not busy fixing what other people broke, you get to work on making new
 stuff. Which some knucklehead will probably, eventually, break too.
 
 Class Primary Attribute: Intelligence
+
+Class Feats: 
+=======
```

Comment:
I'm not sure why 190-192 were added, but as they are redundant and style noncompliant I must ask that they return to the nothingness from whence they came.

Reply:
Will nuke